modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -155,15 +142,10 @@ class Create_Timepoint extends \NDB_Form
         foreach (\Utility::toArray($visitLabelSettings) as $visitLabel) {
             if ($visitLabel['@']['subprojectID']==$this->subprojectID) {
                 if ($visitLabel['generation'] == 'user') {
-                    $this->addBasicText(
-                        'visitLabel',
-                        'Visit label',
-                        array(
-                         'size'      => $visitLabel['length'] + 1,
-                         'maxlength' => $visitLabel['length'],
-                        )
+                    throw new \DeprecatedException(
```

Comment:
I think for 17.1 we should throw a warning or something that this will be deprecated in 18.0 with suggestions of how to upgrade to sequence generation